src/docs/patterns/buttons.mdx

```diff
@@ -0,0 +1,379 @@
+        `DONT: Do not use FABs when the body content is empty. You should use an empty state component instead.`,
+    ]}
+/>
+
+## Colors
+
+For actions that require special attention, such as halting and deleting, consider coloring the button to alert the user.
+
+<ImageGrid images={[RedButton]} captionsUnderImages={[`DO: Use red colors in buttons for destructive actions.`]} />
+
+## Shapes
+
+The default shape of a Brightlayer UI application is square with rounded corners of radius 4dp. Alternatively, consumer-facing applications may also consider using round / pill-shaped buttons consistently throughout the entire product.
+
+<MaterialDesignDescription
+    title={'Project Identity § Component Variants'}
+    url={'/design/project-identity#component-variants'}
+    description={`Learn about style variations to make your project stand out better.`}
+    avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+/>
+
+<ImageGrid
+    images={[ShapeRoundedSquare, ShapeRound]}
+    captionsUnderImages={[
+        `Default Brightlayer UI button styles`,
+        `Round / pill-shaped buttons are only used in consumer-facing products.`,
+    ]}
+/>
+
+## Toggle Button Groups
+
+Functionally related buttons may be grouped together. These buttons share a common container and behave like tab navigations and radio buttons where only one button is active at a time.
+
+<ImageGrid
+    images={[ToggleText, ToggleIcon]}
+    captionsUnderImages={[
+        `A toggle button group where users can select a data category to examine in detail.`,
+        `AVOID: Avoid using icon buttons inside toggle button groups, unless the meaning of the icon can be clearly interpreted from the context, which must be verified with user testing.`,
+    ]}
+/>
+
+# Usage
+
+## Buttons inside Another Component
+
+Buttons are often used within other components.
+
+<ImageGrid
+    images={[InsideEmptyState, InsideCard, InsideBanner, InsideSnackbar]}
+    captionsUnderImages={[
+        `A button inside an empty state encourages users to take action to make the screen less “empty”.`,
+        `Button on a dashboard.`,
+        `Two buttons inside a banner.`,
+        `A button inside a snackbar for quick actions.`,
+    ]}
+/>
+
+## Placement
+
+People generally read from left to right and from top to bottom. You should place buttons in the order you want them to be read.
+
+<ImageGrid
+    regularWidth
+    images={[PlacementChronological]}
+    captionsUnderImages={[
+        `Actions such as “add”, “create”, and “edit” come before actions such as “export” or “delete”. `,
+    ]}
+/>
+
+<ImageGrid
+    images={[PlacementDeemphasize, PlacementCancel]}
+    captionsUnderImages={[
+        `“Skip for Now”, being the least encouraged action, is placed on the bottom.`,
+        `“Cancel” buttons are usually placed to the left of affirmative actions such as “next”, “ok” and “done.”`,
```

Comment:
“done.”` - Full stop should be outside ""

Reply:
the full stop goes inside quotes when it's the end of a sentence